nixos/modules/services/networking/dnsmasq.nix

```diff
@@ -53,6 +53,15 @@ in
         '';
       };
 
+      package = lib.mkOption {
```

Comment:
`mkPackageOptionMD` can be used to simplify this

Reply:
Updated